<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="shortcutBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QTabWidget" name="dirTabs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>30</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab_1">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>0</height>
         </size>
        </property>
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuCreateNew">
     <property name="title">
      <string>Create new...</string>
     </property>
     <addaction name="actionNewDir"/>
     <addaction name="separator"/>
     <addaction name="actionNewFile"/>
    </widget>
    <addaction name="actionNewTab"/>
    <addaction name="actionNewWindow"/>
    <addaction name="separator"/>
    <addaction name="menuCreateNew"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionShowHiddenFiles"/>
   </widget>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>Go</string>
    </property>
    <addaction name="actionUp"/>
    <addaction name="actionBack"/>
    <addaction name="actionForward"/>
    <addaction name="actionHome"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuGo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionNewTab">
   <property name="icon">
    <iconset theme="tab-new"/>
   </property>
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="statusTip">
    <string>Open a new tab within this window.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="icon">
    <iconset theme="window-new"/>
   </property>
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="statusTip">
    <string>Open a new window attached to this instance.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="exit"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="icon">
    <iconset theme="go-up"/>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
   <property name="statusTip">
    <string>Open the parent directory.</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset theme="go-previous"/>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="statusTip">
    <string>Return to the previously visited folder.</string>
   </property>
   <property name="shortcut">
    <string>Back, Alt+Left</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset theme="go-next"/>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="statusTip">
    <string>Undo the last &quot;Back&quot; action.</string>
   </property>
   <property name="shortcut">
    <string>Forward, Alt+Right</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset theme="go-home"/>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="statusTip">
    <string>Open your home directory.</string>
   </property>
   <property name="shortcut">
    <string>Alt+Home</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut"/>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="statusTip">
    <string>Prepare selected file(s) to be moved upon pasting elsewhere.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Prepare selected file(s) to be copied upon pasting elsewhere.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste"/>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste file from clipboard. Action depends on contents of clipboard.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset theme="edit-find"/>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="statusTip">
    <string>Search for files, starting from the current directory.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="reload"/>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="statusTip">
    <string>Reload the current directory.</string>
   </property>
   <property name="shortcut">
    <string>F5, Ctrl+R</string>
   </property>
  </action>
  <action name="actionNewDir">
   <property name="icon">
    <iconset theme="folder-new"/>
   </property>
   <property name="text">
    <string>Directory</string>
   </property>
   <property name="statusTip">
    <string>Create a new directory within the current directory.</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>Empty File</string>
   </property>
   <property name="statusTip">
    <string>Create a new empty file within the current directory.</string>
   </property>
  </action>
  <action name="actionShowHiddenFiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Hidden Files</string>
   </property>
   <property name="statusTip">
    <string>Whether to hide or show the files your system says should be hidden.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
